tracing-subscriber/src/subscribe.rs

```diff
@@ -586,6 +586,9 @@ where
             return outer;
         }
 
+        // The intention behind calling `inner.register_callsite()` before the if statement
+        // was to ensure that the inner subscriber is informed that the callsite exists
```

Comment:
nit: in this context i might use present tense here:
```suggestion
        // is to ensure that the inner subscriber is informed that the callsite exists
```